opencti-platform/opencti-front/src/private/components/common/files/import_files/ImportFilesDialog.tsx

```diff
@@ -544,7 +545,7 @@ const ImportFiles = ({ open, handleClose }: ImportFilesDialogProps) => {
 
 const ImportFilesDialog = ({ open, entityId, handleClose, initialFreeTextContent }: ImportFilesDialogProps) => {
   const initialValue = initialFreeTextContent
-    ? { entityId, activeStep: 1, mode: 'manual', initialFreeTextContent }
+    ? { entityId, activeStep: 1, importMode: 'manual' as ImportMode, initialFreeTextContent }
```

Comment:
I think we could remove as ImportMode if we do that; 

```suggestion
  const initialValue: InitialValues = initialFreeTextContent
    ? { entityId, activeStep: 1, importMode: 'manual', initialFreeTextContent }
    : { entityId };
```

and export interface InitialValues in ImportFilesContext

Reply:
thanks :) it's fixed